test/integration/bootstrap/bootstrap_test.go

```diff
@@ -15,20 +15,114 @@
+			"mt-logs",
+			"af-logs",
+			"fs-logs",
+		}
+		for _, bucket := range bucketReposLogs {
+			urlBucket := fmt.Sprintf("https://www.googleapis.com/storage/v1/b/bkt-%s-%s", projectID, bucket)
+			opBucket := gcloud.Run(t, fmt.Sprintf("storage ls --buckets gs://bkt-%s-%s", projectID, bucket), gcloudArgsBucket).Array()
+			assert.Equal(urlBucket, opBucket[0].Get("metadata.selfLink").String(), fmt.Sprintf("The bucket name should be %s.", urlBucket))
+			assert.True(opBucket[0].Exists(), "Bucket %s should exist.", urlBucket)
+		}
+
+		bucketRepoState := []string{
+			"tf-state",
+		}
+		for _, bucket := range bucketRepoState {
+			urlBucket := fmt.Sprintf("https://www.googleapis.com/storage/v1/b/bkt-%s-%s", projectID, bucket)
+			opBucket := gcloud.Run(t, fmt.Sprintf("storage ls --buckets gs://bkt-%s-%s", projectID, bucket), gcloudArgsBucket).Array()
+			assert.Equal(urlBucket, opBucket[0].Get("metadata.selfLink").String(), fmt.Sprintf("The bucket name should be %s.", urlBucket))
+			assert.True(opBucket[0].Exists(), "Bucket %s should exist.", urlBucket)
+		}
+
+		repos := []string{
+			"eab-applicationfactory",
+			"eab-fleetscope",
+			"eab-multitenant",
+		}
+		for _, repo := range repos {
+			url := fmt.Sprintf("https://source.developers.google.com/p/%s/r/%s", projectID, repo)
+			repoOP := gcloud.Runf(t, "source repos describe %s --project %s", repo, projectID)
+			repoSa := fmt.Sprintf("serviceAccount:tf-cb-%s@%s.iam.gserviceaccount.com", repo, projectID)
+			repoIamOpts := gcloud.WithCommonArgs([]string{"--flatten", "bindings", "--filter", "bindings.role:roles/viewer", "--format", "json"})
+			repoIamPolicyOp := gcloud.Run(t, fmt.Sprintf("source repos get-iam-policy %s", repo), repoIamOpts).Array()[0]
+			listMembers := utils.GetResultStrSlice(repoIamPolicyOp.Get("bindings.members").Array())
+			assert.Contains(listMembers, repoSa, fmt.Sprintf("Service Account %s should have role roles/viewer on repo %s", repoSa, repo))
+			assert.Equal(url, repoOP.Get("url").String(), "source repo %s should have url %s", repo, url)
+		}
+
+		triggerRepos := []string{
+			"eab-applicationfactory",
+			"eab-fleetscope",
+			"eab-multitenant",
+		}
+
+		branchesRegex := `^(development|non\\-production|production)$`
+
+		for _, triggerRepo := range triggerRepos {
+			for _, filter := range []string{
+				fmt.Sprintf("trigger_template.branch_name='%s' trigger_template.repo_name='%s' AND name='%s-apply'", branchesRegex, triggerRepo, triggerRepo),
+				fmt.Sprintf("trigger_template.branch_name='%s' trigger_template.repo_name='%s' AND name='%s-plan'", branchesRegex, triggerRepo, triggerRepo),
+			} {
+				cbOpts := gcloud.WithCommonArgs([]string{"--project", projectID, "--filter", filter, "--format", "json"})
+				cbTriggers := gcloud.Run(t, "beta builds triggers list", cbOpts).Array()
+				assert.Equal(1, len(cbTriggers), fmt.Sprintf("cloud builds trigger with filter %s should exist", filter))
+			}
+		}
+
+		serviceAccounts := []string{
```

Comment:
`tf-cb-eab` is also constant.
And maybe can also get the values from variable `repo` (line 74)